lib/minimal_pipeline/cloudformation.rb

```diff
@@ -80,17 +80,17 @@ def stack_output(stack, output)
     # @param stack_parameters [Hash] Parameters to be passed into the stack
     def deploy_stack(stack_name, stack_parameters)
       unless @client.describe_stacks(stack_name: stack_name).stacks.empty?
-        puts 'Updating the existing stack'
+        puts 'Updating the existing stack' if ENV['DEBUG']
```

Comment:
Are these really debug messages?   I like them all the time

Reply:
Then enable `DEBUG=true` in your pipelines. This is aimed to keep rspec test output clean.